make/modules/jdk.compiler/Gendata.gmk

```diff
@@ -36,58 +40,53 @@ $(eval $(call IncludeCustomExtension, modules/jdk.compiler/Gendata.gmk))
 $(eval $(call ReadImportMetaData))
 
 # Modules that should be visible for 9 - the documented modules:
```

Comment:
While this is an old comment, it seems outdated. I assume "9" refers to "JDK 9", but that is no longer current. Maybe you can rephrase the comment while you're at it?